src/main/java/frc/lib/util/swerveUtil/COTSFalconSwerveConstants.java

```diff
@@ -0,0 +1,184 @@
+package frc.lib.util.swerveUtil;
```

Comment:
Every file that we bring in largely unchanged from another code base needs to have a link to that code base at the top of the file.  That way we can distinguish between code we've written vs code we've incorporated, and if there is any question about how the code works we can go back to the original source.

Reply:
Yes, that's where the file originated from.